### DISCUSSION TITLE
Blacklist channels

### DISCUSSION THREAD

**Discussion by Lex041** [jagrosh/Vortex#130] [Questions] [answered]
Hello! Is it possible to blacklist channels from the tracking of deleted or modified messages? THANKS!

**Comment by jagrosh** [comment, marked as answer]
This is explained on the wiki: 
https://github.com/jagrosh/Vortex/wiki/Rarely-Asked-Questions#how-to-prevent-message-logs-from-a-specific-channel

**Reply by Lex041** [reply]
Thank you!
